app/elements/challenge/challenge-modal.html

```diff
@@ -69,7 +70,17 @@
 			onChallengeResponse: function (res) {
 				this.fire('challengeCreated');
 				page.redirect('/challenges/' + res.detail.response.Key);
+			},
+			_computeDuration: function(){
+				var duration = 0;
+				for (var i = 0; i < this.selectedTasks.length; i++){
+					duration += this.selectedTasks[i].Duration;
+				}
+				this.Duration = duration / 6 / 1e10;
```

Comment:
Why not use the [`Duration`](http://godoc.org/github.com/coduno/api/model#Assignment) of the `Challenge`?